### ISSUE TITLE
Phoenix 1.2.5 does not provide :conn on phoenix_controller_render :start

### ISSUE THREAD

**Issue by zolrath** [timberio/timber-elixir#238]
Hi!
I've just added Timber to a Phoenix 1.2.5 application and the Phoenix Instrumenter does not seem very happy. When I perform an API call to my Phoenix application Ecto logs properly and I get my response but the Phoenix Instrumentation fails. For each request instead of information on the request I get two logged errors:

```
Instrumenter Timber.Integrations.PhoenixInstrumenter.phoenix_controller_render/3 failed.
** (FunctionClauseError) no function clause matching in Timber.Integrations.PhoenixInstrumenter.phoenix_controller_render/3
    (timber) lib/timber/integrations/phoenix_instrumenter.ex:324: Timber.Integrations.PhoenixInstrumenter.phoenix_controller_render(:start, %{application: :phoenix, file: "c:/Users/furd/Documents/GitHub/leadsbyme_api/deps/phoenix/lib/phoenix/controller.ex", function: "do_render/4", line: 646, module: Phoenix.Controller}, %{format: "json", template: "demos.json"})
    (leadsbyme_web) lib/endpoint.ex:1: Leadsbyme.Web.Endpoint.instrument/4
    (phoenix) lib/phoenix/controller.ex:646: Phoenix.Controller.do_render/4
    (leadsbyme_web) lib/controllers/demo_controller.ex:1: Leadsbyme.Web.DemoController.action/2
    (leadsbyme_web) lib/controllers/demo_controller.ex:1: Leadsbyme.Web.DemoController.phoenix_controller_pipeline/2
    (leadsbyme_web) lib/endpoint.ex:1: Leadsbyme.Web.Endpoint.instrument/4
    (leadsbyme_web) lib/phoenix/router.ex:261: Leadsbyme.Web.Router.dispatch/2
    (leadsbyme_web) lib/router.ex:1: Leadsbyme.Web.Router.do_call/2
    (leadsbyme_web) lib/endpoint.ex:1: Leadsbyme.Web.Endpoint.phoenix_pipeline/1
    (leadsbyme_web) lib/plug/debugger.ex:123: Leadsbyme.Web.Endpoint."call (overridable 3)"/2
    (leadsbyme_web) lib/endpoint.ex:1: Leadsbyme.Web.Endpoint.call/2
    (plug) lib/plug/adapters/cowboy/handler.ex:15: Plug.Adapters.Cowboy.Handler.upgrade/4
    (cowboy) c:/Users/furd/Documents/GitHub/leadsbyme_api/deps/cowboy/src/cowboy_protocol.erl:442: :cowboy_protocol.execute/4

Instrumenter Timber.Integrations.PhoenixInstrumenter.phoenix_controller_render/3 failed.
** (FunctionClauseError) no function clause matching in Timber.Integrations.PhoenixInstrumenter.phoenix_controller_render/3
    (timber) lib/timber/integrations/phoenix_instrumenter.ex:324: Timber.Integrations.PhoenixInstrumenter.phoenix_controller_render(:stop, 0, :ok)
    (leadsbyme_web) lib/endpoint.ex:1: Leadsbyme.Web.Endpoint."-instrument/4-after$^3/0-0-"/2
    (leadsbyme_web) lib/endpoint.ex:1: Leadsbyme.Web.Endpoint.instrument/4
    (phoenix) lib/phoenix/controller.ex:646: Phoenix.Controller.do_render/4
    (leadsbyme_web) lib/controllers/demo_controller.ex:1: Leadsbyme.Web.DemoController.action/2
    (leadsbyme_web) lib/controllers/demo_controller.ex:1: Leadsbyme.Web.DemoController.phoenix_controller_pipeline/2
    (leadsbyme_web) lib/endpoint.ex:1: Leadsbyme.Web.Endpoint.instrument/4
    (leadsbyme_web) lib/phoenix/router.ex:261: Leadsbyme.Web.Router.dispatch/2
    (leadsbyme_web) lib/router.ex:1: Leadsbyme.Web.Router.do_call/2
    (leadsbyme_web) lib/endpoint.ex:1: Leadsbyme.Web.Endpoint.phoenix_pipeline/1
    (leadsbyme_web) lib/plug/debugger.ex:123: Leadsbyme.Web.Endpoint."call (overridable 3)"/2
    (leadsbyme_web) lib/endpoint.ex:1: Leadsbyme.Web.Endpoint.call/2
    (plug) lib/plug/adapters/cowboy/handler.ex:15: Plug.Adapters.Cowboy.Handler.upgrade/4
    (cowboy) c:/Users/furd/Documents/GitHub/leadsbyme_api/deps/cowboy/src/cowboy_protocol.erl:442: :cowboy_protocol.execute/4

```

Thanks for your help!

**Comment by DavidAntaramian** [comment]
Hey @zolrath What version of the `timber` package are you running? This looks like it might be a duplicate of #230 which was fixed in v2.5.6+ and v2.6.0+

**Comment by zolrath** [comment]
My `mix.lock` file verifies timber v2.6.0!

**Comment by DavidAntaramian** [comment]
I see the issue. Phoenix 1.2.5 does not provide the `:conn` on render start. I'll fix our integration logic to handle that situation

**Comment by DavidAntaramian** [comment]
Hey @zolrath! The fix for this is in master now, at commit 21b329c1e92449f5c6a1fd6e204cc8ba37e44979. I'm going to cut a new version for you so you can upgrade easily

**Comment by DavidAntaramian** [comment]
Version 2.6.1 is now out and should fix your issue 🎉 

Please reopen if it doesn't
